src/web/components/SlotBodyEnd.tsx

```diff
@@ -6,24 +6,54 @@ const wrapperMargins = css`
     margin: 18px 0;
 `;
 
-export const SlotBodyEnd = () => {
-    const endpointUrl = 'https://contributions.guardianapis.com/epic';
+type Props = {
+    contentType: string;
+    sectionName?: string;
+    shouldHideReaderRevenue: boolean;
+    isMinuteArticle: boolean;
+    isPaidContent: boolean;
+    tags: TagType[];
+};
 
-    const trackingParams = {
-        ophanPageId: window.guardian.config.ophan?.pageViewId,
-        ophanComponentId: 'ACQUISITIONS_EPIC',
-        platformId: 'GUARDIAN_WEB',
-        campaignCode: 'gdnwb_copts_memco_remote_epic_test_api',
-        abTestName: 'remote_epic_test',
-        abTestVariant: 'api',
-        referrerUrl: window.location.href.split('?')[0],
+export const SlotBodyEnd = ({
+    contentType,
+    sectionName,
+    shouldHideReaderRevenue,
+    isMinuteArticle,
+    isPaidContent,
+    tags,
+}: Props) => {
+    // Putting together the request payload
+    const contributionsPayload = {
+        tracking: {
+            ophanPageId: window.guardian.config.ophan.pageViewId,
+            ophanComponentId: 'ACQUISITIONS_EPIC',
+            platformId: 'GUARDIAN_WEB',
+            campaignCode: 'gdnwb_copts_memco_remote_epic_test_api',
+            abTestName: 'remote_epic_test',
+            abTestVariant: 'api',
+            referrerUrl: window.location.href.split('?')[0],
```

Comment:
Could this code ever end up being run on the server? If so, we might need to guard against window being undefined

Reply:
Good question - nothing stopping a developer from dropping the component in a SSR component tree, even through they wouldn't get much value out out it. My preference is to ensure the component doesn't break if used server-side but without having to implement server-side analogous logic, which right now we don't need.
I've added the optional chaining operator to `window?.` so no exceptions in the component - only window specific data would be sent as `undefined` to the API, which feels ok considering we don't always have a server-side way of accessing that data (e.g. Ophan ID).